contracts/Timelock.sol

```diff
@@ -104,6 +111,23 @@ contract InstaTimelock {
         return returnData;
     }
 
+    function executePayload(address target, string memory signature, bytes memory data) public returns (bytes memory) {
```

Comment:
Not sure if ever needed but Aave has this functionality payable https://github.com/aave/arc-timelock/blob/main/contracts/timelock/TimelockExecutorBase.sol#L200

Reply:
In delegateCall is there a use of msg.value?

Reply:
Use-case could be transferring ETH to the contract that is then used in the delegateCall, as the ETH in msg.value will be located on the Timelock contract itself. It will not pass forward to delegateCall target, but the delegateCall implementation still has the ETH available for any action :thinking:

Reply:
Yess!

Reply:
Can skip maybe? Anyway, timelock contract won't hold any ETH mostly

Reply:
Sure let's skip, if it really would be needed for some reason there is always a way